python_modules/dagster-graphql/dagster_graphql/schema/roots/query.py

```diff
@@ -584,13 +589,30 @@ def resolve_assetNodes(self, graphene_info, **kwargs):
         )
 
         depended_by_loader = CrossRepoAssetDependedByLoader(context=graphene_info.context)
+
+        if repo is not None:
+            repos = [repo]
+        else:
+            repos = []
+            used = set()
```

Comment:
Seems like we're never adding anything to this `used` set?

Reply:
good catch